.github/workflows/lynx.yml

```diff
@@ -0,0 +1,58 @@
+name: '🔎 Lynx Testnet Example'
+
+on:
+  schedule:
+    - cron: "0/20 * * * *"  # Every 20 minutes
+  pull_request:
+    branches:
+      - main
+      - development
+      - epic-*
+  workflow_dispatch:
+
+env: # TODO: Use variables when GH supports it for forks. See https://github.com/orgs/community/discussions/44322
+  DEMO_L1_PROVIDER_URI: "https://goerli.infura.io/v3/3747007a284045d483c342fb39889a30"
+  DEMO_L2_PROVIDER_URI: "https://polygon-mumbai.infura.io/v3/3747007a284045d483c342fb39889a30"
+
+jobs:
+  lynx-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ "3.10" ]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install latest Rust stable
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+
+      - id: setup_python
+        name: Set up Python ${{ matrix.python-version }} Environment
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - id: python_cache
+        name: Retrieve Cached Python Dependencies
+        uses: actions/cache@v3
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ runner.os }}-pip-${{ steps.setup_python.outputs.python-version }}-${{ hashFiles('dev-requirements.txt', 'requirements.txt') }}
+
+      - name: Upgrade pip
+        if: steps.python_cache.outputs.cache-hit != 'true'
+        run: python -m pip install --upgrade pip
+
+      - name: Install packages (incl. updated code)
+        run: pip install .[dev]
+
+      - name: Check CLI Entrypoint
+        run: nucypher --help
+
+      - name: Check Python Entrypoint
+        run: python -c "import nucypher; print(nucypher.__version__)"
+
+      - name: Lynx testnet script
+        run: python examples/testnet_simple_taco.py
```

Comment:
Side note: we could think about using `testnet_compound_multichain_taco` instead. A bit of a heavier script because of the conditions. I'm currently noticing some inconsistent behaviour with it, see #3281.

Reply:
Yes, instead or in addition, after all is free!